tools/ci_build/github/azure-pipelines/templates/c-api-cpu.yml

```diff
@@ -649,7 +649,7 @@ jobs:
         sudo apt autoremove
         PATH=/usr/lib/jvm/jre-1.8.0-openjdk.x86_64/bin:${PATH}
         echo "Java Version"
-        java --version
+        java -version
```

Comment:
>java -version [](start = 8, length = 13)

did this fail before? should we have something like "set -e" for the script?

Reply:
Yes, failed and ignored. 

We should add "set -e". I will address it in another PR if you don't mind.